tasks/eth/008-fp-upgrade/VALIDATION.md

```diff
@@ -0,0 +1,62 @@
+  **After**: `0x000000000000000000000000e5965ab5962edc7477c8520243a95517cd252fa9`
+  **Meaning**: Sets the [DisputeGameFactory slot](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.4.0-rc.4/packages/contracts-bedrock/src/L1/SystemConfig.sol#L76). Verify that the `Key` matches the expected slot.
+
+- **Key**: `0xe52a667f71ec761b9b381c7b76ca9b852adf7e8905da0e0ad49986a0a6871815`
+  **Before**: `0x000000000000000000000000dfe97868233d1aa22e815a266982f2cf17685a27`
+  **After**: `0x0000000000000000000000000000000000000000000000000000000000000000`
+  **Meaning**: Clears the old and unused [L2OutputOracle slot](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/src/L1/SystemConfig.sol#L63). Verify that the `Key` matches the expected slot.
+
+
+### `0xbeb5fc579115071764c7423a4f12edde41f106ed` (`OptimismPortalProxy`)
+
+Links:
+- [Etherscan](https://etherscan.io/address/0xbEb5Fc579115071764c7423A4f12eDde41f106Ed)
+- [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/52d3dbd1605dd43f419e838584abd0ec163d462b/superchain/extra/addresses/mainnet/op.json#L8)
+
+Upgrades the `OptimismPortal` to the `OptimismPortal2` implementation.
+
+State Changes:
+- **Key:** [`0x0000000000000000000000000000000000000000000000000000000000000038`](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.4.0-rc.4/packages/contracts-bedrock/snapshots/storageLayout/OptimismPortal2.json#L80C1-L85C5) <br/>
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After**: `0x000000000000000000000000e5965ab5962edc7477c8520243a95517cd252fa9` <br/>
+  **Meaning**: Sets the `DisputeGameFactoryProxy` address in the proxy storage. Consult the [gov proposal](https://gov.optimism.io/t/final-protocol-upgrade-7-fault-proofs/8161) for the proxy addreess value.
+One can verify that this change was made by running `cast call 0xbeb5fc579115071764c7423a4f12edde41f106ed 'disputeGameFactory()(address)' https://ethereum.publicnode.com`
+
+- **Key:** [`0x000000000000000000000000000000000000000000000000000000000000003b`](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.4.0-rc.4/packages/contracts-bedrock/snapshots/storageLayout/OptimismPortal2.json#L101C1-L113C5) <br/>
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After**: `0x000000000000000000000000000000000000000000000000664f90d500000000` <br/>
+  **Meaning**: Sets the `respectedGameType` and `respectedGameTypeUpdatedAt` slot.
+The `respectedGameType` is 32-bits wide at offset 0 which should be set to 0 (i.e. [`CANNON`](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.4.0-rc.4/packages/contracts-bedrock/src/dispute/lib/Types.sol#L28)).
+The `respectedGameTypeUpdatedAt` is 64-bits wide and offset by `4` on that slot. It should be equivalent to the unix timestamp of the block the upgrade was executed.
```

Comment:
can we give a cast command (or other command) for converting those 4 bytes to a timestamp?